app/views/shared/_navbar.html.erb

```diff
@@ -28,6 +28,16 @@
             <%= link_to 'Log out', destroy_employee_session_path, method: :delete, :onclick => "return confirm('Are you sure you want to Log out?')", class: 'btn btn-outline-danger' %>
           </li>
 
+        <% elsif admin_signed_in? %>
```

Comment:
This should go to another (admin) navbar - in my understanding on the main page we are just checking if user is logged in